### PR TITLE
 Simplify calls to collect-logs.sh 

### PR DESCRIPTION
**Pull request by khaledhosny**

### CODE REVIEW

**Comment by ebraminio** [comment]
While the script is on `.ci/` folder, this is a good idea ✅.

The working directory will be reset on each `- run:` command, probably this is the issue with most of the bots.

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,7 +42,10 @@ jobs:
       - checkout
       - run: dnf install -y pkg-config ragel gcc gcc-c++ automake autoconf libtool make which glib2-devel freetype-devel cairo-devel libicu-devel gobject-introspection-devel graphite2-devel redhat-rpm-config python || true
       - run: NOCONFIGURE=1 ./autogen.sh --with-freetype --with-glib --with-gobject --with-cairo --with-icu --with-graphite2
-      - run: mkdir build && cd build && ../configure && make && (make check || (.ci/collect-logs.sh && false))
+      - run: mkdir build && cd build
+      - run: ../configure
```
Comment (ebraminio) [inline .circleci/config.yml]: Working directory won't be preserved thus better to keep this on one line.

File: .circleci/config.yml [hunk 2]
```diff
@@ -9,7 +9,10 @@ jobs:
       - checkout
       - run: apt update && apt install -y ninja-build binutils libtool autoconf automake make cmake gcc g++ pkg-config ragel gtk-doc-tools libfreetype6-dev libglib2.0-dev libcairo2-dev libicu-dev libgraphite2-dev python python-pip
       - run: pip install fonttools
-      - run: ./autogen.sh && (make distcheck || (cd harfbuzz-* && ../.ci/collect-logs.sh && false)) && rm -rf harfbuzz-*
+      - run: ./autogen.sh
+      - run: make
```
Comment (ebraminio) [inline .circleci/config.yml]: I think `make` before would be unnecessary or better to test `distcheck` before running a `make`.

**Comment by behdad** [comment]
Do "make distcheck" before "make" first. That shouldn't fail. We want to test for that.

File: .circleci/config.yml (hunk 1 above)
Comment (khaledhosny) [reply .circleci/config.yml]: Didn’t know that. Fixed.

File: .circleci/config.yml (hunk 2 above)
Comment (khaledhosny) [reply .circleci/config.yml]: Plain `make` can fail, but there is no point in trying to show the test logs for such failure.

**Comment by behdad** [comment]
Let's finish and merge this.

**Comment by khaledhosny** [comment]
I think I’m done now.

**Comment by behdad** [comment]
Thanks. Please merge after bots are happy.

**Pull request merged**